# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/ipc/PDocumentChannel.ipdl
# Commit: abd9cd77f3cb
# Full Hash: abd9cd77f3cb3e6a7734af2b8e4a22de4b931382
# Author: Jean-Yves Avenard <jyavenard@mozilla.com>
# Date: 2020-06-09 15:16:49
# Regressor Bug: 1641737
# File Overlap Count: 1
# Description:
#   Bug 1641737 - P5. Remove unnecessary method. r=mattwoodrow
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D78210
# ==============================================================================

diff -r e3bf9a45db6a -r abd9cd77f3cb netwerk/ipc/PDocumentChannel.ipdl
--- a/netwerk/ipc/PDocumentChannel.ipdl	Tue Jun 09 07:23:57 2020 +0000
+++ b/netwerk/ipc/PDocumentChannel.ipdl	Tue Jun 09 07:24:04 2020 +0000
@@ -52,11 +52,6 @@
   // and notifies the listener via a redirect to the new channel.
   async RedirectToRealChannel(RedirectToRealChannelArgs args, Endpoint<PStreamFilterParent>[] aEndpoint)
       returns (nsresult rv);
-
-  // Tell child to delete channel (all IPDL deletes must be done from child to
-  // avoid races: see bug 591708).
-  async DeleteSelf();
-
 };
 
 } // namespace net
